pym/bob/cmds/build.py

```diff
@@ -833,6 +869,8 @@ def commonBuildDevelop(parser, argv, bobRoot, develop):
         else:
             parser.error("Malformed define: "+define)
 
+    startTime = time.time()
```

Comment:
is this the right place? It excludes bob's runtime from duration print... :thinking: